src/device/usbd.h

```diff
@@ -79,6 +79,8 @@ static inline bool tud_connect(void)
   return true;
 }
 
+void tud_edpt_close(uint8_t rhport, uint8_t ep_addr);
```

Comment:
please remove this, edpt_close should not be exported to application, it will be handled by the usbd and dcd only.

Reply:
Moved.